cleanrl/ppo_rnd_envpool.py

```diff
@@ -242,7 +242,10 @@ def update(self, rews):
 
 
 if __name__ == "__main__":
-    args = parse_args()
+    args = tyro.cli(Args)
+    args.batch_size = int(args.num_envs * args.num_steps)
+    args.minibatch_size = int(args.batch_size // args.num_minibatches)
+    args.num_iterations = args.total_timesteps // args.batch_size
```

Comment:
`args.num_iterations` does not appear to replace the previous `num_updates`. Defined but not used.